src/content/ui/adaptive-responsive/capabilities.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Capabilities & policies
+description: >-
+  Learn how to adapt your app to the
+  capabilities and policies required
+  by the platform, app store, your company,
+  and so on.
+---
+
+Most real-world apps deal with capabilities and
+policies. This page contains advice for how to
+handle that in your code.
+
+## Design to the strengths of each device type
+
+Consider the unique strengths and weaknesses of different devices.
+Beyond their screen size and inputs, such as touch, mouse, keyboard,
+what other unique capabilities can you leverage?
+Flutter enables our code to _run_ on different devices
+but strong design is more than just running code.
+Think about what each platform does best and
+see if there are unique capabilities to leverage.
+
+For example: Apple's App Store and Google’s Play Store
+have different rules that apps need to abide by.
+Different host operating systems have differing
+capabilities across time as well as each other. 
+
+Another example is leveraging the web's extremely
+low barrier for sharing. If you're deploying a web app,
+decide what deep links to support,
+and design the navigation routes with those in mind.
+
+Flutter's recommended pattern for handling different
+behavior based on these unique capabilities is to create
+a set of `Capability` and `Policy` classes for your app.
+
+### Capabilities
+
+A _capability_ defines what the code or device _can_ do.
+Examples of capabilities are: the existence of an API,
+OS-enforced restrictions,
+physical hardware requirements (like a camera). 
+
+### Policies
+
+A _policy_ defines what the code _should_ do.
+Examples of policies include: App store guidelines,
+design preferences, assets or copy that refers
+to the host device, features enabled on the server side. 
+
+### How to structure policy code 
+
+The simplest mechanical way is `Platform.isAndroid`,
+`Platfrom.isIOS`, and `kIsWeb`. These APIs mechanically
+let you know where the code is running but have some
+problems as the app expands where it runs and
+as host platforms add functionality. 
```

Comment:
I think there's a typo here but I can't tell what's it trying to say ("...and as...")